syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -647,3 +647,21 @@ def combine(self, *pointers):
         ps.append(self)
 
         return syft.combine_pointers(*ps)
+
+    def large_prec(self, *args, **kwargs):
```

Comment:
do we need a method to explicitly convert this or should this happen automatically when calling something like fix prec

Reply:
Not needed

Reply:
I wanted this to be separated until we had a full working solution.
Eventually `fix_prec()` will check the precision and use `FixedPrecisionTensor` or `LargePrecisionTensor`.

Reply:
I would make the case then that this should be a class method on large_prec tensor which we can either call manually or add to fix_prec later.

Reply:
The code now decided whether to use LPT or FPT depending on the requested precision.